src/QtLocationPlugin/GoogleMapProvider.cpp

```diff
@@ -78,19 +79,27 @@ void GoogleMapProvider::_googleVersionCompleted() {
     }
 #endif
 
-    QRegExp reg(QStringLiteral("\"*https?://mt\\D?\\d..*/vt\\?lyrs=m@(\\d*)"), Qt::CaseInsensitive);
-    if (reg.indexIn(html) != -1) {
-        _versionGoogleMap = QString(QStringLiteral("m@%1")).arg(reg.capturedTexts().value(1, QString()));
+    static const QRegularExpression regMap( "\"*https?://mt\\D?\\d..*/vt\\?lyrs=m@(\\d*)", QRegularExpression::CaseInsensitiveOption );
+    const QRegularExpressionMatch matchMap = regMap.match( html );
+    if( matchMap.hasMatch() )
+    {
```

Comment:
FYI: The QGC coding style is to not use an extra line for all of these these. They should be on the line above.

Reply:
Sorry I switch between working on QGC and FreeRTOS a lot, and that was FreeRTOS coding style